database-test-data/UserSchema-data-set2.json

```diff
@@ -0,0 +1,5 @@
+{"name": "jaimie diemer", "username":"jaimied", "pass":"Test1234", "email":"jaimied@fake.com", "subscriptions":[
+  {"name":"Netflix","id":"netflix","isSelected":true,"cost":17.99},
+  {"name":"Amazon Prime Video","id":"amazon","isSelected":false,"cost":16.99},
+  {"hulu": "Hulu", "id": "hulu", "isSelected": true, "cost": 12.99}
+], "currentId": 10139 }
```

Comment:
Cool. You can also add new users to your local using the signup from the sign in/signup form Taniel created. To do that, go to your Chrome dev tools > Application tab > Local Storage > sessionToken (right click to delete).